sql-plugin/src/main/scala/com/nvidia/spark/rapids/DeviceMemoryEventHandler.scala

```diff
@@ -18,6 +18,7 @@ package com.nvidia.spark.rapids
 
 import java.io.File
 import java.lang.management.ManagementFactory
```

Comment:
file needs copyrights updated.

Reply:
forgot to set `SPARK_RAPIDS_AUTO_COPYRIGHTER=ON` :)